src/transformStateWithClones.js

```diff
@@ -7,7 +7,32 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const eachAction = [];
```

Comment:
The name `eachAction` could be improved to reflect that it contains the history of states. A name like `stateHistory` or `states` would be more descriptive and align with the naming convention suggested in the checklist.